web_widget_digitized_signature/static/src/js/digital_sign.js

```diff
@@ -7,23 +7,42 @@ openerp.web_widget_digitized_signature = function(instance) {
     instance.web.form.widgets.add('signature', 'instance.web.form.FieldSignature');
         instance.web.form.FieldSignature = instance.web.form.FieldBinaryImage.extend({
         template: 'FieldSignature',
+        placeholder: "/web/static/src/img/placeholder.png",
+        initialize_content: function() {
+            this._super();
+            this.$el.find("#signature").empty().jSignature({'decor-color' : '#D1D0CE', 'color': '#000', 'background-color': '#fff'});
+            this.$el.find("#signature").attr({"tabindex": "0",'height':"100"});
+            this.empty_sign = this.$el.find("#signature").jSignature("getData",'image');
+            this.$el.find('#sign_clean').click(this.on_clear_sign);
+            this.$el.find('.save_sign').click(this.on_save_sign);
+        },
+        on_clear_sign: function() {
+            if (this.get('value') !== false) {
```

Comment:
I suggest you to directly check for the value.
 
if(this.get('value') && !_.isUndefined(value)){
// your code here.
}